public/pages/home/home.html

```diff
@@ -23,4 +23,4 @@
       </div>
     </div>
   </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
```suggestion
</html>

```
ファイル末尾は改行を入れてほしいです。（慣習的に）